crates/ruff_linter/src/rules/flake8_pytest_style/rules/helpers.rs

```diff
@@ -19,6 +19,19 @@ pub(super) fn get_mark_decorators(
     })
 }
 
+pub(super) fn get_parametrize_decorators(
+    decorators: &[Decorator],
+) -> impl Iterator<Item = &Decorator> {
+    decorators.iter().filter_map(|decorator| {
+        let name = UnqualifiedName::from_expr(map_callable(&decorator.expression))?;
+        let ["pytest", "mark", "parametrize"] = name.segments() else {
+            return None;
+        };
+
+        Some(decorator)
+    })
```

Comment:
We could write this slightly more tersely as

```suggestion
    decorators.iter().filter(|decorator| {
        UnqualifiedName::from_expr(map_callable(&decorator.expression))
            .is_some_and(|name| matches!(name.segments(), ["pytest", "mark", "parametrize"]))
    })
```

At that point, I wouldn't really mind just inlining this into `check_test_function_args` since the helper isn't called anywhere else, but I don't feel strongly about either part of this suggestion.